lib/environment_light.rb

```diff
@@ -11,5 +11,12 @@
 require 'bundler/setup'
 
 # Load application settings
-require File.expand_path("../../config/initializers/rails_config", __FILE__)
-Settings = RailsConfig.load_files ["settings.yml","settings.local.yml", "settings/#{Rails.env}.yml"].map{|f| "#{Rails.root}/config/#{f}" }
+require Rails.root.join('config/initializers/rails_config.rb')
+Settings = RailsConfig.load_files([
+    'settings.yml',
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.